src/containers/ClassifierContainer.jsx

```diff
@@ -107,10 +109,16 @@ class ClassifierContainer extends React.Component {
             )}
             <button href="#" className="white-red button">Your Crib Sheet</button>
             <img className="divider" role="presentation" src={Divider} />
+            
+            
+            <button href="#" className="white-green button" onClick={this.prepareSubmitClassificationForm}>Done</button>
+            
+            {/*
             <button href="#" className="white-green button" onClick={this.completeClassification}>Done</button>
             <button href="#" className="green button" onClick={this.submitClassificationAndRedirect}>
               Done &amp; Talk
             </button>
```

Comment:
Same here on the commenting out.

Reply:
Why have the buttons got href attributes? Those should only appear on links.